src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -25,7 +27,7 @@ const logger = createLoggerWithLabel(module)
  */
 export const handleSubmitFeedback: RequestHandler<
   { formId: string },
-  unknown,
+  ErrorDto | { message: string; isPageFound: boolean; formTitle: string },
```

Comment:
nit: prefer if the return type is typed to something like `PrivateFormErrorDto`

Reply:
i agree - waiting on anty's review on [this pr](https://github.com/opengovsg/FormSG/pull/1572/files), which has the dto and will fix once the other pr is merged.